particles/Tutorial/Kotlin/README.md

```diff
@@ -100,8 +101,9 @@ Meanwhile, the Kotlin looks quite different, as this is where the magic occurs:
 package arcs.tutorials
```

Comment:
There is a reference to npm start on line 74/75, can this be updated to tools/sigh devServer?

Reply:
(Also should be updated in particles/Tutorial/Javascript/README.md)